ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -349,6 +349,14 @@ objects:
                         --key /tmp/artifacts/installer/tls/journal-gatewayd.key \
                         --url "https://${bootstrap_ip}:19531/entries?_SYSTEMD_UNIT=${service}.service"
                 done
+                if ! whoami &> /dev/null; then
+                  if [ -w /etc/passwd ]; then
+                    echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+                  fi
+                fi
+                ssh-add /tmp/cluster/ssh-privatekey
```

Comment:
also you need to run `eval $(ssh-agent)` to run the agent and setup the env

Reply:
Do we even need to use the agent? 
Can you use `-i` to pass in the key?

Reply:
> Can you use `-i` to pass in the key?

That covers the CI -> bootstrap hop, but you need the agent for subsequent bootstrap -> control-plane hops.

Reply:
I see.